torch/csrc/jit/script/resolver.h

```diff
@@ -34,13 +34,12 @@ struct Resolver {
   virtual std::shared_ptr<SugaredValue> resolveValue(
       const std::string& name,
       Function& m,
-      const SourceRange& loc) const {
+      const SourceRange& loc) {
```

Comment:
I think it is reasonable that a Resolver may change its internal state when resolveValue/resolveType is called. For SourceImporter, they are not constant: the importer is parsing and processing files and definitions when this stuff is called, changing its internal state. I removed it here rather than const-casting in source importer because I think this API more accurately reflects the contract that a resolver can be stateful.